.github/workflows/go.yml

```diff
@@ -0,0 +1,36 @@
+name: Go
```

Comment:
I'll make sure to disable this workflow since it's specific to the upstream repo.

Reply:
I can take the workflow out if you prefer. Not sure that we need it at all